src/sentry/incidents/subscription_processor.py

```diff
@@ -492,6 +504,137 @@ def process_results_workflow_engine(
             )
         return results
 
+    def process_legacy_metric_alerts(
```

Comment:
fyi - this is a noop, only moved this code into a method and return the list of incident triggers.